main.go

```diff
@@ -110,13 +112,45 @@ func (u *urlsStruct) home(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func urlsFromJSON(u *urlsStruct, f string) {
```

Comment:
2 things here:

 - it seems to me that the function could be declared as a method of `urlsStruct` and be named `LoadURL` for example.
 - this should return an error, not call `log.Fatalf()`. The error should be managed by the caller, maybe the caller will want to exit the application, maybe not, but this decision should be the caller responsibility.